### ISSUE TITLE
Color contrast in result area is too low

### ISSUE THREAD

**Issue by jdeisenberg** [codekiln/gradus-reason#27]
The color contrast is sufficiently low to be difficult to read; it fails WCAG guidelines. (You can see this if you load the attached image in https://contrastchecker.com/ )
![gradus2](https://user-images.githubusercontent.com/160096/37171010-5b772b3e-22c1-11e8-8b9f-7a30161406c2.png)

**Comment by codekiln** [comment]
Thank you for the bug report and for pointing to WCAG and contrastchecker, this is easy to fix.
